src/connector/src/source/base.rs

```diff
@@ -169,6 +184,32 @@ pub enum SplitReaderImpl {
     GooglePubsub(Box<PubsubSplitReader>),
 }
 
+pub enum FsSplitReaderImpl {
```

Comment:
Why defining `FsSplitReaderImpl` in a separate enum? 🤔

Reply:
`FsSplitReaderImpl` impl a different reader trait.